10.detect-separation/scripts/nbconverted/2.apply-mycn-signature.py

```diff
@@ -0,0 +1,123 @@
+
+# coding: utf-8
+
+# # Applying MYCN Signature Learned in TARGET data to Cell Lines
+# 
+# **Gregory Way, 2019**
+
+# In[1]:
+
+
+import os
+import sys
+import matplotlib.pyplot as plt
+import seaborn as sns
+import numpy as np
+import pandas as pd
+
+sys.path.append("../8.gtex-interpret")
+from scripts.utils import load_weight_matrix, apply_signature
+
+
+# In[2]:
+
+
+np.random.seed(123)
+
+
+# In[3]:
+
+
+get_ipython().run_line_magic('matplotlib', 'inline')
+
+
+# ## 1. Load NBL Processed Dataset
+
+# In[4]:
+
+
+file = os.path.join('data', 'nbl_celllines_processed_matrix.tsv.gz')
+nbl_df = pd.read_table(file, index_col=0)
+
+print(nbl_df.shape)
+nbl_df.head(2)
+
+
+# ## 2. Apply Signature Learned by TARGET data
+
+# In[5]:
+
+
+vae_seed = '451283'
+vae_k = 200
+vae_feature = "vae_111"
```

Comment:
Why is this hard-coded in?  I'm not sure how this signature is selected?

Reply:
It is selected from the `1.separate` module. But it is a good idea to remove the hard code! I will draw it in from a file generated in `1.separate` instead. Thanks!